src/main/java/com/napier/sem/App.java

```diff
@@ -43,11 +43,11 @@ public static void main(String[] args) {
 
         // Use-case 2.1-2.3
         // Run top N countries query against database
-        // ArrayList<Country> countries = dao.TopNCountriesIn(WORLD, "", 10);
+        // ArrayList<Country> countries = dao.topNCountriesIn(WORLD, "", 10);
```

Comment:
Small piece of refactoring to keep it uniform with the rest of the methods. Same for changes to DAO.